LDKNodeMonday/Service/Lightning Service/LightningNodeService.swift

```diff
@@ -50,7 +54,7 @@ class LightningNodeService {
         config.logLevel = .trace
 
         let nodeBuilder = Builder.fromConfig(config: config)
-        nodeBuilder.setChainSourceEsplora(serverUrl: storedEsploraURL, config: nil)
+        nodeBuilder.setChainSourceEsplora(serverUrl: storedServer!.url, config: nil)
```

Comment:
would like to avoid `!` where possible, any other ways you can think of doing this (and `backupInfo!` a few lines above) so we dont get a crash if it hits these?